openfisca_survey_manager/scenarios.py

```diff
@@ -1597,6 +1597,12 @@ def init_variable_in_entity(simulation, entity, variable_name, series, period):
         )
 
     if enum_variable_imputed_as_enum:
+        if series.isnull().any():
+            log.debug('There are {} NaN values for {} non NaN values in variable {}'.format(
```

Comment:
```suggestion
            log.debug('There are {} NaN values ({}% of the array) in variable {}'.format(
```

Minor and technically equivalent, but might be slightly nicer.

Reply:
We need also to change the next lines right @nikhilwoodruff ?

Reply:
Yes, commented.